src/config/unit_test.rs

```diff
@@ -189,7 +201,7 @@ impl UnitTest {
             if let Some((inputs, outputs)) = results.get(&check.extract_from) {
                 if check.conditions.is_empty() {
                     inspections.push(format!(
-                        "check transform '{}' payloads (events encoded as JSON):\n{}\n{}",
+                        "check transform {:?} payloads (events encoded as JSON):\n{}\n{}",
```

Comment:
We're using `Debug` quite a bit for error messages and I'm not sure it's the best choice. How about we stick with `Display` and add an explicit implementation for `ComponentId` so that it's a bit more clear this is user-facing?